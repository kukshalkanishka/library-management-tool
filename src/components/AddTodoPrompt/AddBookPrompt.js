import React from "react";

const AddBookPrompt = () => {
  return (
      <form method="POST" action="/addBook" className="addTodo">
      <input
        type="text"
        name="book"
        placeholder="Add your book here"
        className="addTodoBox"
        autocomplete="off"
        required
      />
      <input type="submit" value="ADD BOOK" className="addBookButton" />
    </form>
  );
};

export default AddBookPrompt;
