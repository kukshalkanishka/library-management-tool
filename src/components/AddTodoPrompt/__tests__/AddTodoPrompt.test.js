import React from "react";
import { shallow, configure } from "enzyme";
import AddBookPrompt from "../AddBookPrompt";

import Adapter from "enzyme-adapter-react-16";

configure({ adapter: new Adapter() });

describe("AddBookPrompt", () => {
  it("should render properly", () => {
    const wrapper = shallow(<AddBookPrompt />);
    expect(wrapper.debug()).toMatchSnapshot();
  });
});
