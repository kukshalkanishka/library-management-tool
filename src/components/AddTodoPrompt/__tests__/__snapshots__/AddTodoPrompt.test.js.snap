// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddTodoPrompt should render properly 1`] = `
"<form method=\\"POST\\" action=\\"/addTodo\\" className=\\"addTodo\\">
  <input type=\\"text\\" name=\\"todo\\" placeholder=\\"Add your todo here\\" className=\\"addTodoBox\\" autocomplete=\\"off\\" required={true} />
  <input type=\\"submit\\" value=\\"ADD TODO\\" className=\\"addTodoButton\\" />
</form>"
`;
